t/esn-journalnewcomment.t

```diff
@@ -194,7 +194,7 @@ test_esn_flow(sub {
     $LJ::CAP{0...15}->{track_thread} = 0;
     $subsc->delete;
 
-    if (LJ::Event::JournalNewComment->zero_journalid_subs_means eq "friends") {
+    if ( ( LJ::Event::JournalNewComment->zero_journalid_subs_means // "undef" ) eq "friends") {
```

Comment:
... because I thought that was winding up in an error message if that was a test failure! I wanted it to be a nice failure message ( was undef expected friends ) if it was undef.  Can fix!